ipv8/messaging/lazy_payload.py

```diff
@@ -171,9 +175,7 @@ def from_unpack_list(cls, %s):
     return compile(f_code, f_code, 'exec')
 
 
-def _compile_to_pack_list(src_cls: typing.Type[VariablePayload],
-                          format_list: typing.List[FormatListType],
-                          names: typing.List[str]) -> types.CodeType:
+def _compile_to_pack_list(src_cls: VT, format_list: List[FormatListType], names: List[str]) -> types.CodeType:
```

Comment:
I also changed the import of typing to the `from typing import ...` form to make the function signature fit into a single line